docs/manual/EINK_ERROR_CODES.md

```diff
@@ -0,0 +1,14 @@
+# AmpliPro E-Ink Display Error Codes
+
+The following numerical codes are displayed on the AmpliPro's E-Ink Display when the corresponding error occurs. Note that these codes do not appear on older AmpliPi/AmpliPro devices that use TFT touchscreen displays.
```

Comment:
Is there a reason we're not supporting showing error codes on older units? Most of the units in the field have TFT displays at this point.

Reply:
As of this pull request there still isn't a field showing error codes on TFT displays. That sounds like it is outside of the scope of this pull request, but it would make sense to add at some point.

Reply:
I discussed with Ryan and we will keep it out of this PR but it will be the next task.